src/test/Makefile.am

```diff
@@ -8,8 +8,7 @@ AM_LDFLAGS = \
 AM_CPPFLAGS = \
 	-I$(top_srcdir) \
 	-I$(top_srcdir)/src/include \
-	-I$(top_builddir)/src/common/libflux \
-	$(ZMQ_CFLAGS)
+	-I$(top_builddir)/src/common/libflux
```

Comment:
There are no more c targets in src/test, so AM_CPPFLAGS, AM_LDFLAGS, AM_CFLAGS, and LDADD can all go away.